lib/rules/unit-disallowed-list/README.md

```diff
@@ -157,3 +157,40 @@ The following patterns are considered problems:
 ```css
 @media print and (max-resolution: 100dpi) {}
 ```
+
+### `ignoreFunctions: ["function", "/regex/", /regex/]|"function"|"/regex/"|/regex/`
+
+Ignore units that are inside of the specified functions.
+
+For example, with `["px"]`.
+
+Given:
+
+```json
+{
+  "ignoreFunctions": ["calc", "/^translate/"]
+}
```

Comment:
[suggest] Considering consistency with other secondary options in this doc, it seems better to omit the option name:

```suggestion
["calc", "/^translate/"]
```